src/compass.js

```diff
@@ -46,6 +47,9 @@ function clearCompass() {
 
 function draw(what) {
   console.log("Call: Compass.draw");
+  compass.backgroundColor(Themes.currentTheme().textColour);
```

Comment:
Is there a point to updating the colour on each draw or can we move this into init?